docs/kpod-rm.1.md

```diff
@@ -0,0 +1,31 @@
+% kpod(1) kpod-rm - Remove one or more containers
+% Ryan Cole
+# kpod-rm "1" "August 2017" "kpod"
+
+## NAME
+kpod rm - Remove one or more containers
+
+## SYNOPSIS
+**kpod** **rm** [*options* [...]] container
+
+## DESCRIPTION
+Kpod rm will remove one or more containers from the host.  The container name or ID can be used.  This does not remove images.  Running containers will not be removed without the -f option
+
+## OPTIONS
+
+**--force, f**
```

Comment:
If we're keeping the --volume and --link flags, they should be listed in the manpage

Reply:
But, the --volume and --link flags are hidden. The user wouldn't know about its existence unless they look into the code. I don't think it should be added to the manpage yet.

Reply:
I agree, lets remove the flags, I am not sure they make any sense with what we have done so far,  If kpod run adds support for builtin volumes(Not sure I want it, then we could revisit removal of them).  I have no idea what the --link flag is doing.  @runcom Any idea?

```
man docker rm
...
       -l, --link=true|false
          Remove the specified link and not the underlying container. The
       default is false.
```

Reply:
Fixed.